recipes/evalhyd-cli/meta.yaml

```diff
@@ -0,0 +1,55 @@
+# name for conda package (i.e. `conda install -c conda-forge <project_name>`)
+{% set project_name = "evalhyd-cli" %}
+# name for command
+{% set command_name = "evalhyd" %}
+{% set version = "0.1.0.0" %}
+{% set sha256 = "38ebcc4915542b0887d4178724643539355b043a7aa4a68bdeacce4629bd1f11" %}
+
+package:
+  name: {{ project_name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.irstea.fr/HYCAR-Hydro/evalhyd/{{ project_name }}/-/archive/v{{ version }}/{{ project_name }}-v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
```

Comment:
Since all packages seem to be built from the same sources, it would make sense IMO to build everything in one multi-output recipe.

That way the dependence of (e.g.) the python bindings on the C++ lib is also much clearer and less fragile.

If you don't know how to do this, there are many examples (e.g. [arrow](https://github.com/conda-forge/arrow-cpp-feedstock/blob/main/recipe/meta.yaml)), or I can try to help.

Reply:
Thank you very much for your review @h-vetinari 

I was not sure which way to go. And I tried with separate recipes, because I used different versions for the different packages (e.g. _0.1.0_ for the C++ core lib, and _0.1.0_.0 for the Python/R bindings), i.e. I keep an additional number for small changes to the bindings not affecting the C++ core lib.

If I had a multi-output recipe, would that prevent me from having separate versions?

Reply:
> If I had a multi-output recipe, would that prevent me from having separate versions?

If each source release is consistent, the versioning of the individual components is secondary. And yes, it's possible, see [scipopt](https://github.com/conda-forge/scipoptsuite-feedstock/blob/main/recipe/meta.yaml) for an extreme example (all components have a separate version). BTW: that's a case where variables are warranted. ;-)

Reply:
OK, thank you.

And when you say that my packages are built fron the same sources, they come from the same GitLab project, but they are in different repositories. Is this a problem?

Reply:
Ah, actually, I overlooked that with the templated variables, apologies. In that case, the multiple recipes are fine actually.

Please just use the strings directly, rather than a fake variable.

Reply:
Ok, understood. I have removed jinja templating where possible.